hack/man-page-checker

```diff
@@ -40,7 +40,7 @@ done
 #
 # Make sure the descriptive text in podman-foo.1.md matches the one
 # in the table in podman.1.md.
-for md in *.1.md;do
+for md in *-*.1.md;do
```

Comment:
@edsantiago WDYT?

Reply:
This prevents podman.1.md itself from being included in the checks for parent descriptions